src/app/services/answer.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { Observable, from } from 'rxjs';
+import { filter } from 'rxjs/operators';
+
+import { Answer } from '../models/answer';
+import { ANSWERS } from '../mock-data/mock-answers';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AnswerService {
+  private _randomId: number = Math.floor(Math.random() * 10) + 1;
```

Comment:
Just a recommendation, if the value isn't ever going to change, you can mark it as `readonly`:

`private readonly _randomId: number = ...`